sdk/purview/ci.yml

```diff
@@ -79,10 +73,6 @@ extends:
         groupId: com.azure
         safeName: azureanalyticspurviewadministration
         releaseInBatch: ${{ parameters.release_azureanalyticspurviewadministration }}
-      - name: azure-analytics-purview-catalog
-        groupId: com.azure
-        safeName: azureanalyticspurviewcatalog
-        releaseInBatch: ${{ parameters.release_azureanalyticspurviewcatalog }}
```

Comment:
purview team seems forgot to clean up the ci.yml, after they removed `azure-analytics-purview-catalog`